app/test/ui/widgets/firebase_auth_mock.dart

```diff
@@ -0,0 +1,52 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
```

Comment:
Can we have BSD license with our GPL? We need to watch what code we use in our project. There's probably a tool that will get all the licenses for our plugins but we'll need to manually the license for code we use like this.

- [ ] Verify we can use this code and track the license somewhere

For example, we'll have an `about` button in the side drawer that will list all the licenses we used in our software. For further example, see our university webmail. There is an `about` button in the bottom-left corner that shows what we should do if we have multiple licenses.